elasticdl/python/tests/test_helper.py

```diff
@@ -0,0 +1,176 @@
+from elasticdl.python.worker.worker import Worker
+
+
+def create_recordio_file(size, dataset, shape):
+    temp_file = tempfile.NamedTemporaryFile(delete=False)
+    with closing(recordio.Writer(temp_file.name)) as f:
+        for _ in range(size):
+            if dataset == "imagenet":
+                image = np.random.randint(255, size=shape, dtype=np.uint8)
+                image = tf.image.encode_jpeg(tf.convert_to_tensor(value=image))
+                image = image.numpy()
+                label = np.ndarray([1], dtype=np.int64)
+                label[0] = np.random.randint(1, 11)
+                example_dict = {
+                    "image": tf.train.Feature(
+                        bytes_list=tf.train.BytesList(value=[image])
+                    ),
+                    "label": tf.train.Feature(
+                        int64_list=tf.train.Int64List(value=[label])
+                    ),
+                }
+            elif dataset == "frappe":
+                feature = np.random.randint(5383, size=(shape,))
+                label = np.random.randint(2, size=(1,))
+                example_dict = {
+                    "feature": tf.train.Feature(
+                        int64_list=tf.train.Int64List(value=feature)
+                    ),
+                    "label": tf.train.Feature(
+                        int64_list=tf.train.Int64List(value=[label])
+                    ),
+                }
+            elif dataset == "test_module":
+                x = np.random.rand(shape).astype(np.float32)
+                y = 2 * x + 1
+                example_dict = {
+                    "x": tf.train.Feature(
+                        float_list=tf.train.FloatList(value=x)
+                    ),
+                    "y": tf.train.Feature(
+                        float_list=tf.train.FloatList(value=y)
+                    ),
+                }
+            elif dataset == "image_default":
+                image = np.random.rand(np.prod(shape)).astype(np.float32)
+                label = np.ndarray([1], dtype=np.int64)
+                label[0] = np.random.randint(0, 10)
+                example_dict = {
+                    "image": tf.train.Feature(
+                        float_list=tf.train.FloatList(value=image)
+                    ),
+                    "label": tf.train.Feature(
+                        int64_list=tf.train.Int64List(value=[label])
+                    ),
+                }
+            else:
+                raise ValueError("Unknown dataset name %s." % dataset)
+
+            example = tf.train.Example(
+                features=tf.train.Features(feature=example_dict)
+            )
+            f.write(example.SerializeToString())
+    return temp_file.name
+
+
+def distributed_train_and_evaluate(
+    feature_shape,
+    model_zoo_path,
+    model_def,
+    model_params="",
+    training=True,
+    dataset="image_default",
+    callback_classes=[],
+    use_async=False,
+    get_model_steps=1,
+):
```

Comment:
Add some documentation on this function, especially on what datasets are available. Otherwise developers have to look into the function to see what's available.

Reply:
Done.

Reply:
Done.